terraform/environments/electronic-monitoring-data/modules/landing_bucket/outputs.tf

```diff
@@ -1,3 +1,7 @@
 output "bucket_id" {
   value = module.this-bucket.bucket.id
+}
+
+output "bucket_arn" {
```

Comment:
im guessing youve tried the below and it doesnt work?
```suggestion
output "bucket" {
  value = module.this-bucket.bucket
```

Reply:
im not sure the suggestion will acc work if you commit it tho lol

Reply:
not sure what you're angling for here anyway!